CondTools/Geometry/test/hgcalgeometrywriter.py

```diff
@@ -2,7 +2,7 @@
 
 process = cms.Process("HGCalGeometryWriter")
 process.load('CondCore.CondDB.CondDB_cfi')
-process.load('Configuration.Geometry.GeometryExtended2023D3_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D41_cff')
```

Comment:
@kpedro88 @ggovi @ianna these are just examples, then they can be used with whatever scenario compatible with the parameters, am I correct?

Reply:
I just tried to update the script not to refer to a nonexistent geometry.